ShijiLighting/GHIElectronics.TinyCLR.Drivers.ShijiLighting.APA102C/APA102C.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Drawing;
+using GHIElectronics.TinyCLR.Devices.Spi;
+
+namespace GHIElectronics.TinyCLR.Drivers.ShijiLighting.APA102C
+{
+    public class APA102C {
+        private readonly SpiController spiBusContoller;
+        private readonly SpiDevice spiBus;
+        private readonly byte[] startFrame;
+        private readonly byte[] stopFrame;
+        private readonly byte[] ledFrame;
+        private readonly int pixelCount;
+
+        public APA102C(int pixelCount, string spiId, int chipselect) {
```

Comment:
`chipSelect`

Reply:
Chip select is there until the ability to choose the option to not use controller chip select is fixed. The user will need to pass a pin that is never going to be used.

Reply:
I meant the spelling, just need to capitalize the s